docs/guide/search.md

```diff
@@ -365,16 +365,24 @@ Secondly, copy configuration files needed for eZ Solr Search Engine bundle, in t
 
 ``` bash
 # Make sure to replace the /opt/solr/ path with where you have placed Solr
-cp -R vendor/ezsystems/ezplatform-solr-search-engine/lib/Resources/config/solr/* /opt/solr/example/solr/collection1/conf/
-
-/opt/solr/bin/solr start -f 
+cd /opt/solr/example
+mkdir -p multicore/collection1/conf
+cp -R <ezplatform-solr-search-engine>/lib/Resources/config/solr/* multicore/collection1/conf
+cp solr/collection1/conf/{currency.xml,stopwords.txt,synonyms.txt} multicore/collection1/conf
+## Remove default cores configuration and add core configuration
+sed -i.bak 's/<core name=".*" instanceDir=".*" \/>//g' multicore/solr.xml
+sed -i.bak "s/<shardHandlerFactory/<core name=\"collection1\" instanceDir=\"collection1\" \/><shardHandlerFactory/g" multicore/solr.xml
+cp multicore/core0/conf/solrconfig.xml multicore/collection1/conf
+sed -i.bak s/core0/collection1/g multicore/collection1/conf/solrconfig.xml
+
+/opt/solr/bin/solr start -f -a "-Dsolr.solr.home=multicore"
```

Comment:
@andrerom fixed in https://github.com/ezsystems/developer-documentation/pull/22/commits/d0be8a2eee7d55ff41a39eaf3423130950f99c71